app/controllers/api_controller.rb

```diff
@@ -44,6 +44,7 @@ def current_sync_region
   end
 
   def block_level_sync?
+    return false unless current_user
     current_user.block_level_sync? && requested_sync_region_id == current_block.id
```

Comment:
This still exists because some users (telemed officers) are not eligible for block syncing, regardless of the feature flag, right?

Reply:
Yes, just for telemed officers.

Reply:
In that case, can we remove `current_user.block_level_sync?` and replace it with just `!can_teleconsult?` and move that condition into the `current_sync_region` function? If `block_level_sync` is being deprecated as a concept, with this PR, we should see all instances of `block_level_sync` in the codebase gone.

Reply:
We will continue to do both kind of syncs even after this PR. The `block_level_sync` concept still exists _and_ is tied to a user.  I avoided putting the logic in the controller since the selection criteria for block sync is a bit arbitrary (can the user teleconsult?) and subject to change.

As an alternative, we add a `district_level_sync?` on `User`, since block level sync is the default now. Added in c2295bca, wdyt?